lib/isafw/isaplugins/ISA_cve_plugin.py

```diff
@@ -161,17 +171,21 @@ def process_report_type(self, rtype):
             flog.write("Args: " + args)
         try:
             popen = subprocess.Popen(
-                args, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-            stdout_value = popen.communicate()[0]
+                args, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            result = popen.communicate()
         except:
-            stdout_value = "Error in executing cve-check-tool"
+            tool_stderr_value = "Error in executing cve-check-tool" + str(sys.exc_info())
             with open(self.logfile, 'a') as flog:
                 flog.write("Error in executing cve-check-tool: " +
-                           sys.exc_info())
+                           str(sys.exc_info()))
         else:
-            report = self.report_name + "." + rtype
-            with open(report, 'wb') as freport:
-                freport.write(stdout_value)
+            stdout_value = result[0]
+            tool_stderr_value = result[1]
+            if not tool_stderr_value :
```

Comment:
@pohly, Does this look like a good fix: https://github.com/01org/meta-security-isafw/commit/346706b8e3b19e2fe22c5aba8de8d3ad05ef9089